packages/dear-imgui/cpp/src/element/element.cpp

```diff
@@ -24,6 +24,42 @@ std::unique_ptr<Element> Element::makeElement(const json& nodeDef, ReactImgui* v
 
     if (nodeDef.is_object() && nodeDef.contains("style") && nodeDef["style"].is_object()) {
         element->m_layoutNode->ApplyStyle(nodeDef["style"]);
+
+        if (nodeDef["style"].contains("backgroundColor") || nodeDef["style"].contains("borderColor")) {
+            BaseDrawStyle baseDrawStyle;
+
+            if (nodeDef["style"].contains("backgroundColor")) {
+                baseDrawStyle.backgroundColor = jsonHEXATupleToIV4(nodeDef["style"]["backgroundColor"]);
+            }
+
+            if (nodeDef["style"].contains("borderColor")) {
+                baseDrawStyle.borderColor = jsonHEXATupleToIV4(nodeDef["style"]["borderColor"]);
+            }
+
+            if (baseDrawStyle.backgroundColor.has_value() || baseDrawStyle.borderColor.has_value()) {
+
+                if (nodeDef["style"].contains("rounding")) {
+                    baseDrawStyle.rounding = nodeDef["style"]["rounding"].template get<float>();
+                }
+
+                if (nodeDef["style"].contains("borderThickness")) {
+                    baseDrawStyle.borderThickness = nodeDef["style"]["borderThickness"].template get<float>();
+                }
+
+                if (nodeDef["style"].contains("roundCorners") && nodeDef["style"]["roundCorners"].is_array()) {
+                    const auto roundCorners = nodeDef["style"]["roundCorners"].template get<std::vector<std::string>>();
+
+                    baseDrawStyle.drawFlags = std::accumulate(
+                        roundCorners.begin(),
+                        roundCorners.end(),
+                        static_cast<ImDrawFlags>(ImDrawFlags_None),
+                        cornersToDrawFlags
+                    );
+                }
+
+                element->m_baseDrawStyle.emplace(baseDrawStyle);
+            }
+        }
```

Comment:
Thank you for the suggestion. However, please note that neither `nodeDef["style"]["backgroundColor"]` nor `nodeDef["style"]["borderColor"]` are fail-safe. I must ensure `nodeDef["style"]` contains either of them prior to accessing them, as otherwise this may trigger an error.

Reply:
I think I'm going to leave the implementation as is, but thanks anyway.